scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -363,7 +413,7 @@ Function RunDirectMethodAmqpTest
     TestSetup
 
     $testStartAt = Get-Date
-    $deviceId = "e2e-${ReleaseLabel}-Windows-${Architecture}-DMAmqp"
+    $deviceId = "e2e-${ReleaseLabel}-Windows-${Architecture}-DMAmqp-$(Get-Date -UFormat %s)"
```

Comment:
"ReleaseLabel" is expected to be unique, which we are using $(Release.ReleaseName).$(Release.AttemptNumber), like Release-1134.1.  why do you need to add "Date" to construct device id?

Reply:
In the build environment that's true, but in my case I was running the tests repeatedly from the command line. What I should have done is embed the timestamp command into the `-ReleaseLabel` argument on the command line. I'll back out these timestamp changes.